lib/mix/lib/mix/dep/converger.ex

```diff
@@ -48,6 +48,9 @@ defmodule Mix.Dep.Converger do
   """
   def converge(acc, lock, opts, callback) do
     {deps, acc, lock} = all(acc, lock, opts, callback)
+    remote = Mix.RemoteConverger.get
+    # TODO: Remove function_exported? check before releasing 1.4
+    if remote && function_exported?(remote, :post_converge, 0), do: remote.post_converge()
```

Comment:
Actually this is a bad idea. We shouldn't change deps and lock directly because we cannot update the converger accumulator.

Basically we can't do it for the same reason we have to run the converger a second time after the remote has run - there's a risk the deps will say and the accumulator another.